packages/db/src/schema/posts.ts

```diff
@@ -0,0 +1,31 @@
+import {
+	pgTable,
+	text,
+	timestamp,
+	jsonb,
+	primaryKey,
+	boolean,
+} from "drizzle-orm/pg-core";
+import { profiles } from "./profiles.ts";
+
+export const posts = pgTable(
+	"posts",
+	{
+		slug: text("slug").notNull(),
+		locale: text("locale").notNull(),
+		title: text("title").notNull(),
+		description: text("description"),
+		socialImage: text("social_image"),
+		bannerImage: text("banner_image"),
+		version: text("version"),
```

Comment:
`version` should also be part of the PK if we use the `post_data` approach above.

Reply:
How if version is optional?